shell/server/shell-server.js

```diff
@@ -60,6 +60,8 @@ BrowserPolicy.content.allowImageOrigin(staticAssetHost);
 BrowserPolicy.content.allowScriptOrigin(staticAssetHost);
 BrowserPolicy.content.allowFontOrigin(staticAssetHost);
 BrowserPolicy.content.allowConnectOrigin(staticAssetHost);
+BrowserPolicy.content.allowConnectOrigin("wss:");
```

Comment:
Doesn't this allow WebSocket connections to _any_ origin, and if so, isn't that unwanted?